src/components/share/CountDate.js

```diff
@@ -0,0 +1,43 @@
+function CountDate(time){
+  const creat = time // 받아온 time(Card에 적혀있는 시간)
```

Comment:
P3:
매개변수명을 원하시는 이름으로 하시면 구지 `create`에 재할당할 필요 없어보입니다.

```suggestion
function CountDate(createdTime){
```